lib/l10n/app_de.arb

```diff
@@ -0,0 +1,24 @@
+{
+  "about": "Über",
+  "settings": "Einstellungen",
+  "theme": "Design",
+  "system": "System",
+  "dark": "Dunkel",
+  "light": "Hell",
+  "text": "Text",
+  "translation": "Übersetzung",
+  "source": "Quellsprache",
+  "target": "Zielsprache",
+  "languages": "Sprachen",
+  "swap": "Tauschen",
+  "copy": "In die Zwischenablage kopieren",
+  "search": "Suchen...",
+  "history": "Verlauf",
+  "deleteAll": "Alles löschen",
+  "deleteTitle": "Möchten Sie den Verlauf wirklich löschen?",
+  "deleteDescription": "Sie können diese Aktion nicht rückgängig machen.",
+  "clear": "Verlauf leeren",
+  "cancel": "Abbrechen",
+  "error": "Etwas ist schief gelaufen!",
+  "copyToast": "In die Zwischenablage kopiert.",
```

Comment:
"copyToast": "In die Zwischenablage kopiert."

Reply:
fixed